source/beamline/VESPERS/VESPERSEndstation.h

```diff
@@ -111,7 +114,7 @@ public slots:
 	/// Sets the current control that should be focused on.  It will find the correct control based on the name.  Control is set to 0 if invalid name is given.
 	void setCurrent(QString name);
 	/// Toggles the current state of the laser power.
-	void toggleLaserPower() { toggleControl(laserPower_); emit laserPoweredChanged(); }
+	void toggleLaserPower() { laserPower_->move(1); emit laserPoweredChanged(); }
```

Comment:
No, these PVs are poorly designed.  Writing a 1 _toggles_ the state.  If you were at 0, writing a 1 would result in a state of 1.  If you were at state 1, writing a 1 would result in a state of 0.  It's lame because the state PV uses a 0 and 1 as real states.